README.md

```diff
@@ -30,6 +30,10 @@ If you have experience with the repo and are looking for a specific section docu
 - [Golden Tags](docs/entities/golden_tags.md)
 - [Summary Metrics](docs/entities/summary_metrics.md)
 
+In case you are looking for **experimental features**, check out our new features here:
+
+- [Relationship Synthesis](docs/relationship_synthesis.md)
```

Comment:
Only mentioning Relationship Synthesis as the entrypoint, the links to candidates should come from the Relationship Synthesis document. 

Do you think it is a good approach? Something else we would like to add here?